backends-velox/src/test/scala/io/glutenproject/execution/VeloxDataTypeValidationSuite.scala

```diff
@@ -353,7 +353,11 @@ class VeloxDataTypeValidationSuite extends WholeStageTransformerSuite {
     // Validation: BatchScan Project Aggregate Sort Limit
     // TODO validate Expand operator support Struct type ?
     runQueryAndCompare("select int, struct.struct_1 from type1 " +
-      "sort by struct.struct_1 limit 1") { _ => }
+      "sort by struct.struct_1 limit 1") { df => {
```

Comment:
Do we need to add the nested struct type test？

Reply:
There are several tests from Spark on that in [GlutenDataFrameSetOperationsSuite](https://github.com/oap-project/gluten/blob/main/gluten-ut/spark32/src/test/scala/org/apache/spark/sql/GlutenDataFrameSetOperationsSuite.scala) and were fixed.